src/ceph.in

```diff
@@ -259,6 +259,9 @@ def parse_cmdargs(args=None, target=''):
 
     parser.add_argument('-c', '--conf', dest='cephconf',
                         help='ceph configuration file')
+
+    parser.add_argument('-k', '--keyring', dest='keyring',
```

Comment:
@cp542524698 `keyring` is a ceph option, and can always be specified by `ceph.conf` using 
```ini
[client]
        keyring = /path/to/keyring
```

i don't think we need to clutter the help info with the ceph options, even they are need under variant circumstances.